src/index.js

```diff
@@ -175,11 +200,27 @@
         }
         return collapsedChildren;
       };
-      // This method forces the visual cue to be cleared. It is to be called in extreme cases 
+      // This method forces the visual cue to be cleared. It is to be called in extreme cases
       api.clearVisualCue = function(node) {
         cy.trigger('expandcollapse.clearvisualcue');
       };
-      
+
+      api.disableCue = function() {
+        var options = getScratch(cy, 'options');
+        if (options.cueEnabled) {
+          cueUtilities('unbind', cy, api, $);
```

Comment:
'$' sign here and below must be removed, since jQuery dependency is removed from this extension.